src/commands/branches.ts

```diff
@@ -134,6 +138,7 @@ export const builder = (argv: yargs.Argv) =>
           .middleware((args: any) => {
             args.id = args.targetId;
             args.pointInTime = args['source@(timestamp'];
+            args.branchId = args.id; // for analytics
```

Comment:
why did you need a separate line here?

Reply:
This command doesn't have an `id` parameter and doesn't use `branchIdFromProps` either, so we have to treat it separately.

Reply:
oh, right. But `id` property will also be empty, no? you probably need the target-id

Reply:
It's set 2 lines above. I could use `targetId` if you think it's clearer.

Reply:
ahhh, I finally got it! :D